packages/protobuf-test/src/next/perf.ts

```diff
@@ -29,15 +28,15 @@ import {
 /* eslint-disable no-console, import/no-named-as-default-member */
 
 run("Parsing binary", [
-  function () {
-    const data = readFileSync("./descriptorset.binpb");
-    return {
-      name: `large google.protobuf.FileDescriptorSet (${data.byteLength} bytes)`,
-      fn: () => {
-        fromBinary(FileDescriptorSetDesc, data);
-      },
-    };
-  },
+  // function () {
+  //   const data = readFileSync("./descriptorset.binpb");
+  //   return {
```

Comment:
Was this intentional?

Reply:
Yes. Testing a real-world case is good, but the size of this set changes depending on the version of protoc and the proto files, so it's not a good test fixture. We can look into a better replacement that allows for a reliable comparison between v1 and v2.